src/DecodeStream/DecodeStream.cpp

```diff
@@ -820,7 +826,7 @@ try { //Giant try block around all code to get error messages
                                                                                           qMatrix,
                                                                                           pictureHeight, width, chromaFormat);
 
-            //du.stream() >> sliceio::highQualityVBR(slicePrefix, sliceScalar)
+            //inStream >> sliceio::highQualityVBR(slicePrefix, sliceScalar)
```

Comment:
I realise this was already commented out, but do you know why it might have been there? (In either case, now might be a good time to drop this dead code unless it is somehow explanatory in a way I'd not realised?)

Reply:
I don't, cleaning it up sounds like a good idea :)